public/css/widget/state-change.less

```diff
@@ -39,4 +39,69 @@
       .var(background-color, body-bg-color);
     }
   }
+
+  // Avoid transparency on overlapping solid state-change state-balls
+  .state-ball.handled {
+    position: relative;
+    opacity: 1;
+
+    i {
+      position: relative;
+      z-index: 3;
+    }
+
+    &:before {
+      content: "";
+      display: block;
+      position: absolute;
+      top: 0;
+      right: 0;
+      bottom: 0;
+      left: 0;
+      border-radius: 50%;
+      opacity: .6;
+      z-index: 2
+    }
+
+    &:after {
+      content: "";
+      display: block;
+      position: absolute;
+      top: -2px;
+      left: -2px;
+      right: -2px;
+      bottom: -2px;
+      border-radius: 50%;
+      .var(background-color, body-bg-color);
+      z-index: 1;
+    }
+
+    &.state-pending:before {
+      background-color: @color-pending;
```

Comment:
This and the other background colors are not using the css `var` function. (You don't have to use the `.vars` mixin, just `var(--color-pending, @color-pending)` is sufficient.)

Reply:
✅